prime-router/docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Universal Pipeline Technical Documentation 
+
+### Table of Contents
```

Comment:
To clean up the directory structure a bit, what do you think about moving docs_deprecated and generated folders to inside the docs folder instead of inline with it?

Reply:
Now I'm thinking docs/docs_deprecated/generated but we can ask at eng sync for ideas